### PR TITLE
minor edits for clarity

### PR DESCRIPTION
**Pull request by tdykstra**
@cartermp Please review

### CODE REVIEW

**Comment by mairaw** [comment]
Since you're now editing this topic, I'll make some suggestions. Can't add a line note since you haven't touched some of these lines:
- Throughout the topic:
NET CLI -> .NET Core CLI
OSX -> OS X
fbbonacci -> Fibonacci
- Line 5:
read the overview -> read the .NET Core SDK overview
- Line 27: link to the command page like the other commands in the topic are doing
- Line 84: .net core -> .NET Core
- Line 91: remove duplicate from
- Line 103: 
creat -> create
64 bit Windows 10 and the 64 bit version of Mac OSX -> 64-bit version of Windows 10 and the 64-bit version of OS X

**Comment by tdykstra** [comment]
@mairaw done

File: docs/getting-started/cli-console-app-tutorial.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Let's do a quick walkthrough:
 
 1. `$ dotnet new`
 
-   `dotnet new` creates an up-to-date `project.json` file with NuGet dependencies necessary to build a console app.  It also creates a `Program.cs`, a basic file containing the entry point for the application.
+   [`dotnet new`](http://dotnet.github.io/docs/core-concepts/core-sdk/cli/dotnet-new.html) creates an up-to-date `project.json` file with NuGet dependencies necessary to build a console app.  It also creates a `Program.cs`, a basic file containing the entry point for the application.
```
Comment (mairaw) [inline docs/getting-started/cli-console-app-tutorial.md]: the links should be relative paths to those docs

Comment (tdykstra) [inline docs/getting-started/cli-console-app-tutorial.md]: @mairaw changed links to relative

**Comment by mairaw** [comment]
Thanks @tdykstra! LGTM

**Comment by cartermp** [comment]
👍 Thanks!

**Pull request merged**